fe/fe-core/src/main/java/org/apache/doris/catalog/Env.java

```diff
@@ -2890,6 +2890,12 @@ public static void getDdlStmt(DdlStmt ddlStmt, String dbName, TableIf table, Lis
                 sb.append(olapTable.getCompressionType()).append("\"");
             }
 
+            // unique key table with merge on write
+            if (olapTable.getKeysType() == KeysType.UNIQUE_KEYS) {
```

Comment:
if (lapTable.getEnableUniqueKeyMergeOnWrite())

Reply:
if user not set the property, we should not show it.

Reply:
fixed